index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Mobile Portfolio</title>
+  <div class="header-items">
+    
+    <img src="img/My Logo.svg" alt="humburger" class="Logo" > 
+    <img src="img/Union.svg" alt="humburger"  class="Union" >
+  </div>
+
+</head>
+<body>
+  <header>
+    <div class="container">
+      <h2>I'm Edgah.<br>
```

Comment:
- Kindly use an h1 tag here. This is because the h1 tag is the first header tag visible on a page and it is used for the title of a page or post.